### PR TITLE
merge android metadata articles

### PR DESCRIPTION
**Pull request by petekanev**
- merge Android metadata overview and Android metadata Limitations articles into 1

- remove references to `--chrome` as it will become the default tns debug ios option, and `--inspector` is introduced

### CODE REVIEW

File: docs/runtimes/android/advanced-topics/V8-heap-snapshots.md [hunk 1]
```diff
@@ -5,6 +5,17 @@ description: "Using V8 heap snapshots with the Android Runtime"
 position: 3
 ---
 
+> **NOTE**: `nativescript-dev-android-snapshot` plugin is deprecated. The recommended approach for using V8 Heap Snpashot feature in the context of NativeScript is via [`nativescript-dev-webpack` plugin](https://docs.nativescript.org/tooling/bundling-with-webpack#v8-heap-snapshot) in combination with `--snapshot` flag.
```
Comment (sis0k0) [inline docs/runtimes/android/advanced-topics/V8-heap-snapshots.md]: Snpashot -> Snapshot

File: docs/runtimes/android/advanced-topics/V8-heap-snapshots.md [hunk 2]
```diff
@@ -5,6 +5,17 @@ description: "Using V8 heap snapshots with the Android Runtime"
...
+
+Benefits of using `nativescript-dev-webpack` plugin in combination with `--snapshot` flag:
+
+ 1. No limitation for using only a few pre-generated snapshot packages (`tns-core-modules-snapshot` and `nativescript-angular-snapshot`).
```
Comment (sis0k0) [inline docs/runtimes/android/advanced-topics/V8-heap-snapshots.md]: I don't think we have the right context here to start with exactly that point. If I didn't know what the old snapshot plugin does, I wouldn't understand why we're mentioning these pre-generated snapshot packages.

File: docs/runtimes/android/advanced-topics/V8-heap-snapshots.md [hunk 3]
```diff
@@ -5,6 +5,17 @@ description: "Using V8 heap snapshots with the Android Runtime"
...
+ 2. Fine-grained control over what is included in the snapshot. By changing the webpack configuration you can include any module in the snapshotted bundle.
+ 3. An Angular app can take advantage of both ahead of time compilation and heap snapshotting.
```
Comment (sis0k0) [inline docs/runtimes/android/advanced-topics/V8-heap-snapshots.md]: Again, if I didn't know we had that limitation previously, I wouldn't understand why we're mentioning it at all as a benefit.

File: docs/runtimes/android/advanced-topics/V8-heap-snapshots.md [hunk 4]
```diff
@@ -5,6 +5,17 @@ description: "Using V8 heap snapshots with the Android Runtime"
...
+ 4. No need to choose between Webapck and heap snapshot. You can have both.
```
Comment (sis0k0) [inline docs/runtimes/android/advanced-topics/V8-heap-snapshots.md]: Same as above. Also Webapck -> Webpack

File: docs/runtimes/android/advanced-topics/V8-heap-snapshots.md [hunk 5]
```diff
@@ -5,6 +5,17 @@ description: "Using V8 heap snapshots with the Android Runtime"
...
+ 5. It is easier to troubleshoot errors because the whole process of heap snapshot generation is happening on the developer machine.
+ 6. It is more scalable and effective.
```
Comment (sis0k0) [inline docs/runtimes/android/advanced-topics/V8-heap-snapshots.md]: Compared to?

Comment (sis0k0) [reply docs/runtimes/android/advanced-topics/V8-heap-snapshots.md]: Overall, I think we should just remove the 'Usage' section about the old plugin and replace it with a link to the section in the webpack article.

File: docs/runtimes/android/advanced-topics/V8-heap-snapshots.md (hunk 3 above)
Comment (petekanev) [reply docs/runtimes/android/advanced-topics/V8-heap-snapshots.md]: Good point, thought moving it at the top would help readers ignore the previous snapshotting approach, maybe the 'benefits list' should be removed altogether

File: docs/runtimes/android/metadata/overview.md [hunk 6]
```diff
@@ -3,14 +3,50 @@ nav-title: "Overview"
...
+# Metadata API Level
+
+Only Android public APIs (**including those of any plugins added to the project**) present in the metadata will be accessible in JavaScript/TypeScript. That means, if an application is built with metadata for API level 23 (Android Marshmallow 6.0 – 6.0.1), the application user might have problems when running the application on an older device, for example with API levels 17 through 19 (Android KitKat 4.4 – 4.4.4).
+
+Metadata is built automatically for the application. The metadata API level, or simply put, what API level the metadata is built for, is determined by the `--compileSdk` flag passed to the build. By default the nativescript CLI automatically detects the highest Android API level installed on the developer's machine and passes it to the build implicitly. This `--compileSdk` flag can be supplied explicitly when starting a build: `tns run android --compileSdk=1`.
+
+# Metadata Limitations
```
Comment (etabakov) [inline docs/runtimes/android/metadata/overview.md]: There should be only one h1 heading on a single page. So all other headings should use h2.

File: docs/tooling/debugging.md [hunk 7]
```diff
@@ -71,4 +69,12 @@ To debug NativeScript applications in [Visual Studio Code](https://code.visualst
...
-Debugging android apps is as easy as executing `tns debug android`, to debug iOS apps with Chrome DevTools, the `--chrome` flag needs to be passed - `tns debug ios --chrome`. A complete list of features, and how to use them can be found at [Debugging NativeScript apps with Chrome DevTools]({%slug chrome-devtools%})
+Debugging Android and iOS applications is as easy as executing `tns debug <android | ios>`. A complete list of features, and how to use them can be found at [Debugging NativeScript apps with Chrome DevTools]({%slug chrome-devtools%}).
+
+## Debugging iOS Applications with the WebKit Web Inspector
+
+To debug iOS applications using the WebKit Web Inspector debugger the `--inspector` flag needs to be supplied to the `tns debug ios` command.
+
+## Known Problems
+
+- The Chrome DevTools Inspector url is different after each application restart while running a debug session.
```
Comment (etabakov) [inline docs/tooling/debugging.md]: This applies for iOS only, right?

**Review by etabakov — COMMENTED** [review]
Looks very good. Some minors comments from me ...

File: docs/tooling/debugging.md (hunk 7 above)
Comment (petekanev) [reply docs/tooling/debugging.md]: @etabakov that's right, I should specify it in the article if chrome is becoming default.

**Review by Plamen5kov — APPROVED** [review]

**Review by sis0k0 — APPROVED** [review]

**Pull request merged**